### PR TITLE
New package: cxxtest

### PR DESCRIPTION
**Pull request by bryanherman**

### CODE REVIEW

File: var/spack/repos/builtin/packages/cxxtest/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    url      = "https://sourceforge.net/projects/cxxtest/files/cxxtest/4.4/cxxtest-4.4.tar.gz/download"
+
+    version('4.4', sha256='1c154fef91c65dbf1cd4519af7ade70a61d85a923b6e0c0b007dc7f4895cf7d8')
+
+    def install(self, spec, prefix):
+        for path in os.listdir(os.getcwd()):
+            if os.path.isdir(path):
+                shutil.copytree(path, os.path.join(prefix, path))
+            else:
+                shutil.copy(path, os.path.join(prefix, path))
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/cxxtest/package.py]: Hmm, can we be a little more specific than just copying the current working directory into prefix?

**Review by alalazo — CHANGES_REQUESTED** [review]
I'm a bit concerned about the install method. Can we look for some known header, maybe using some function from `llnl.util.filesystem`?

File: var/spack/repos/builtin/packages/cxxtest/package.py (hunk 1 above)
Comment (bryanherman) [reply var/spack/repos/builtin/packages/cxxtest/package.py]: I'll look into a better approach, thanks for the feedback.

Comment (bryanherman) [reply var/spack/repos/builtin/packages/cxxtest/package.py]: I used ``install_tree`` from ``llnl.util.filesystem``. Please take another look when you have time.

**Review by alalazo — APPROVED** [review]

**Pull request merged**